samples/bluetooth/mesh/light/prj.conf

```diff
@@ -29,7 +25,6 @@ CONFIG_BT_LL_SW_SPLIT=y
 CONFIG_BT_OBSERVER=y
 CONFIG_BT_PERIPHERAL=y
 CONFIG_BT_SETTINGS=y
-CONFIG_BT_TINYCRYPT_ECC=y
```

Comment:
`CONFIG_BT_TINYCRYPT_ECC=y` would be set in hci_rpmsg, which is maybe why it wouldn't be needed on the host side in this case?

The defaults haven't been changed that i can recall.

Reply:
This seems to fail in runtime on nRF53 without this line

Reply:
Added again with comment.

Reply:
Need to add them back in the other cases as well. Maybe not needed with a coment, but If commenting anything I would say "Needed for host-only build", since nrf91 would be affected as well for example.

This would not be needed anymore with the changes from https://github.com/nrfconnect/sdk-nrf/pull/3244

Reply:
Removed comment as it is not true when the other PR is merged.